pkg/config/config_template.yaml

```diff
@@ -505,6 +505,49 @@ api_key:
   #
   # validation_period: 60
 
+
+## @param fips - custom object - optional
+## Enter specific configurations for using the FIPS proxy.
+## Uncomment this parameter and the one below to enable them.
+#
+# fips:
+
+  ## @param enabled - boolean - optional - default: false
+  ## @env DD_FIPS_ENABLED - boolean - optional - default: false
+  ## Enable the use of the FIPS proxy to send data to the DataDog backend. Enabling this will force all outgoing traffic
+  ## from the Agent to the local proxy.
+  ##
+  ## When setting this to true the following settings would be overridden, ignoring the values from the
+  ## configuration:
+  ## - dd_url
+  ## - apm_config.apm_dd_url
+  ## - apm_config.profiling_dd_url
+  ## - apm_config.telemetry.dd_url
+  ## - process_config.process_dd_url
+  ## - logs_config.use_http
+  ## - logs_config.logs_no_ssl
+  ## - logs_config.logs_dd_url
+  ## - database_monitoring.metrics.dd_url
+  ## - database_monitoring.activity.dd_url
+  ## - database_monitoring.samples.dd_url
+  ## - network_devices.metadata.dd_url
+  #
+  # enabled: false
+
+  ## @param local_address - string - optional - default: localhost
+  ## @env DD_FIPS_LOCAL_ADDRESS - string - optional - default: localhost
+  ## The local address used by the FIPS proxy to bind ports.
+  #
+  # local_address: localhost
+
+  ## @param port_range_start - integer - optional - default: 3833
+  ## @env DD_FIPS_LOCAL_PORT_RANGE_START - integer - optional - default: 3833
+  ## The FIPS proxy will bind a range of consecutive ports, each dedicated to a specific product (Metrics, Logs, APM.
+  ## Database monitoring, ...). 'port_range_start' is the first port in this range. The Agent will automatically assign
+  ## the right port to the right product based on this information
+  #
+  # port_range_start: 3833
```

Comment:
IANA has this port assigned as [`aipn-auth`](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=73
). as well as subsequent ports.

Was there a specific reason behind this particular number?  Given the trouble we've had with the expvar port ([`commplex-link`](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=88)), could we make a better choice?